examples/bulkProperties-set-request-rebuild.json

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
This file is a hangover from the renaming of example files.

It has been replaced by `bulkProperties-put-request-rebuild-200.json`